invoker/invoker.go

```diff
@@ -124,3 +125,24 @@ func (i *Invoker) Script(height uint64, script []byte, arguments []cadence.Value
 
 	return proc.Value, nil
 }
+
+func (i *Invoker) GetAccount(address flow.Address, header *flow.Header) (*flow.Account, error) {
```

Comment:
Because of the following line:

```go
ctx := fvm.NewContext(zerolog.Nop(), fvm.WithBlockHeader(header))
```

Reply:
But the invoker also has an index, it can get the header from the height, no?

Like for the `Script()` method:
```go
func (i *Invoker) Script(height uint64, script []byte, arguments []cadence.Value) (cadence.Value, error) {

	// Encode the arguments from Cadence values to byte slices.
	var args [][]byte
	// ...

	// Look up the current block and commit for the block.
	header, err := i.index.Header(height)
	if err != nil {
		return nil, fmt.Errorf("could not get header: %w", err)
	}

```